nixos/modules/services/networking/pleroma.nix

```diff
@@ -6,6 +6,12 @@ in {
     services.pleroma = with lib; {
       enable = mkEnableOption "pleroma";
 
+      epmd = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the clustering daemon service. If unprotected, it can be a security issue.";
+      };
+
```

Comment:
```suggestion
      epmd = mkOption {
        type = types.bool;
        default = false;
        description = "Whether to enable the clustering daemon service. If unprotected, it can be a security issue.";
      };

```